css/style.css

```diff
@@ -53,7 +53,20 @@ article img {
   padding-right: 10px;
 }
 .container {
-  position: relative;
-    text-align: center;
-    color: black;
-}
\ No newline at end of file
+    position: relative; 
+    max-width: 800px; /* Maximum width */
+    margin: 0 auto; /* Center it */
+}
+
+.container .content {
+    position: absolute; /* Position the background text */
+    bottom: 0; /* At the bottom. Use top:0 to append it to the top */
+    background: transparent; /* Black background with 0.5 opacity */
+    color: black; /* Grey text */
```

Comment:
A coworker of mine used to say that an out-of-date comment was worse than no comment at all: it gives people the wrong impression if they're coming back to look at the code in the future. It looks like these comments came (I presume) from StackOverflow or similar; make sure to both update/delete those comments as necessary, and give credit to the person/site that helped you solve this problem in your README.